### ISSUE TITLE
Custom Nodes

### ISSUE THREAD

**Issue by bulutharbeli** [Stability-AI/StableSwarmUI#31]
Hello. I am trying to install comfyui manager as usual but it doesn't appear on the panel of swarmui ?
How to fix that ? Thank you

**Comment by mcmonkey4eva** [comment]
Make sure ComfyUI and ComfyUI-Manager are both fully up to date.
Manager was fixed for compatibility just a week ago: <https://github.com/ltdrdata/ComfyUI-Manager/pull/43>
Prior to that fix it was exactly as you said, not showing up on the swarmui page, but now it works fine

**Comment by jlitz** [comment]
I got swarm installed  with comfy as a new install.  there does not seem to be a   way of installing the comfy manager.    I tried installing manager in a  new foldeer  custom_nodes folder,  but  it did not show up in Swarm comfy.   

I  tried to configure the server via  my existing comfy install,  using the self starting script but can not find the  corrects starting scritpt.    I tried this  C:\COMFY\ComfyUI_windows_portable\ComfyUI\main.py   but that produced error not found.   need some help
